kt/plugins/godot-intellij-plugin/src/main/resources/template/.gitignore.template

```diff
@@ -0,0 +1,138 @@
+
+# Created by https://www.toptal.com/developers/gitignore/api/intellij+all,kotlin,gradle
+# Edit at https://www.toptal.com/developers/gitignore?templates=intellij+all,kotlin,gradle
+
+### Intellij+all ###
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff
+.idea/**/workspace.xml
+.idea/**/tasks.xml
+.idea/**/usage.statistics.xml
+.idea/**/dictionaries
+.idea/**/shelf
+
+# Generated files
+.idea/**/contentModel.xml
+
+# Sensitive or high-churn files
+.idea/**/dataSources/
+.idea/**/dataSources.ids
+.idea/**/dataSources.local.xml
+.idea/**/sqlDataSources.xml
+.idea/**/dynamic.xml
+.idea/**/uiDesigner.xml
+.idea/**/dbnavigator.xml
+
+# Gradle
+.idea/**/gradle.xml
+.idea/**/libraries
```

Comment:
Ignore the full `.idea` directory instead of specifying things one by one. You can also use `!` as the prefix - which means don't ignore this <pattern>